src/Lykke.Service.Limitations/Modules/ServiceModule.cs

```diff
@@ -47,8 +48,18 @@ protected override void Load(ContainerBuilder builder)
 
         private void RegisterRepositories(ContainerBuilder builder)
         {
-            builder.Register(ctx => AzureBlobStorage.Create(_appSettings.ConnectionString(s => s.LimitationsSettings.BlobStorageConnectionString))).SingleInstance();
-            builder.Register(ctx => AzureTableStorage<SwiftTransferLimitationEntity>.Create(_appSettings.ConnectionString(s => s.LimitationsSettings.LimitationSettingsConnectionString), "SwiftTransferLimitations", ctx.Resolve<ILogFactory>())).SingleInstance();
+            builder
+                .Register(ctx =>
+                    AzureBlobStorage.Create(_appSettings.ConnectionString(s => s.LimitationsSettings.BlobStorageConnectionString), TimeSpan.FromSeconds(30)))
```

Comment:
This is not the way how we should fix performance problems.  Also, this is not a 'test' branch, not 'hotfix'.